### DISCUSSION TITLE
Lack of isolation between customers

### DISCUSSION THREAD

**Discussion by marwengiggs** [pushsecurity/browser-identity-attacks-matrix#25] [Ideas]
Hi

On lateral movement, you can also add cross tenant attack, access/exfil other customers data 

Best regards

**Comment by jacques-** [comment]
Thanks @marwengiggs! I just saw this guy: https://www.bleepingcomputer.com/news/security/new-microsoft-azure-ad-cts-feature-can-be-abused-for-lateral-movement/ yesterday, is that what you are referring to or possibly another example/different technique you are thinking of?
